src/business.ts

```diff
@@ -114,4 +129,54 @@ export class BusinessApi {
     const success = !resp?.payload?.errorCode
     return { success, body: { ...resp.payload } }
   }
+
+  /*
+   * Function to take the attributes of a new Business account, create that
+   * in Marqeta, and return the account information. If no attributes are
+   * provided, a new Business account will still be created and its
+   * token ID returned.
+   */
+  async create(create: {
+     business: Business
+  }): Promise<{
+    success: boolean,
+    body?: Business,
+    error?: MarqetaError,
+  }> {
+    const searchOptions = snakecaseKeys(create?.business)
+    const resp = await this.client.fire(
+      'POST',
+      'businesses',
+      {},
+      { ...searchOptions }
+    )
+    // catch 404s...
+    if (resp?.payload?.errorCode >='404000' && resp?.payload?.errorCode <= 404999) {
+      const error = resp?.payload?.errorMessage
+      return {
+        success: false,
+        error: {
+          type: 'marqeta',
+          error,
+          status: resp?.payload?.errorCode,
+          marqetaStatus: resp?.payload?.errorCode
+        },
+      }
+    }
+    // catch all other errors...
+    if (resp?.payload?.errorCode > 404999){
+      const error = resp?.payload?.errorMessage
+      return {
+        success: false,
+        error: {
+          type: 'marqeta',
+          error,
+          status: resp?.payload?.errorCode,
+          marqetaStatus: resp?.payload?.errorCode
```

Comment:
In the Peach Client, Peach returned a `status` and the HTTP request had a `status` - that's why there was a `status` and a `peachStatus`... but here? They are the same... why have them?

I can see having the `status` be the HTTP status... like 200, 300, 400, etc. And then the detailed status like 404003 as well... but that's different. Here... they are exactly the same.

And what about the error messages? Wouldn't those be nice to have here? An `errorCode` and `errorMessage` would be very nice - in addition to the `status`.

Right?

Reply:
The `error` is the `response.errorMessage`.  I'll adjust the `return` to match Marqeta's response pattern, versus the Peach response return pattern.